app/i18n/en/gen.php

```diff
@@ -79,6 +79,7 @@
 		'last_year' => 'Last year',
 		'mar' => 'mar',
 		'march' => 'Mar',
+		'may_' => 'May',
```

Comment:
Thanks, should be fixed in https://github.com/FreshRSS/FreshRSS/pull/1617/commits/b80c2ee014827b2fcaa72676263359115a55230a